wazimap/settings.py

```diff
@@ -24,13 +24,16 @@
 DATABASES['default']['ATOMIC_REQUESTS'] = True
 
 INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
```

Comment:
Hey @phillipahereza . How come we've added these?

Reply:
@DavidLemayian shoot, I think the changes in this file were committed in error as I was debugging why the test DB wasn't working. I'll remove them

Reply:
@phillipahereza Cool. Thanks.